docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1283,9 +1283,9 @@ and <<script-fields,Script fields>>.
 end::script-fields[]
 
 tag::scroll-size[]
-The `size` parameter that is used in {es} searches. The default value is `1000`. 
-The maximum value is the value of `index.max_result_window` which is 10,000 by 
-default.
+The `size` parameter that is used in {es} searches when the {dfeed} do not use 
```

Comment:
nit: should it be "... when the {dfeed} **does** not use..."

Reply:
@dimitris-athanasiou Thanks for spotting! Fixed it via 9d20ae6.